components/RuiBreadcrumbs/src/RuiBreadcrumbs.css.ts

```diff
@@ -0,0 +1,97 @@
+import {css} from 'lit-element';
+
+
+/**
+	*  RuiBreadcrumbs Variables
+ 	*  Added colour to prevent empty block
+*/
+
+export const variables = css`
+	:host {
+		/* BASE STYLES */
+
+		/**
+		 * @variable The base colour for the breadcrumbs
+		 */
+		--color: #000000;
+	
+		/**
+		 * @variable Hover colour for the breadcrumbs
+		 */
+		--hover-color: #01447E;
+	
+		/**
+		 * @variable padding size
+		 */
+		--padding: 8px;
+	}
+    `;
+
+/**
+	* RuiBreadcrumbs CSS
+*/
+
+export const layout = css`
+	/**
+		* css for json array breadcrumbs
+	*/
+
+	nav.crumbs ol,
+	nav.childCrumbs ol {
+		list-style: none;
+	}
+	
+	nav.crumbs li,
+	nav.childCrumbs li {
+		display: inline;
+	}
+	
+	nav.crumbs li + li::before {
+		padding: var (--padding);
+		color: var(--color);
+		content: "/";
```

Comment:
Is this different to the "separator" property?

Reply:
Also please refer to my comment below (https://github.com/DeloitteDigitalAPAC/rhythm-ui/pull/13/files#r305687022) regarding not rendering separators as content